charts/ingress-nginx/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: ingress-nginx
 # When the version is modified, make sure the artifacthub.io/changes list is updated
 # Also update CHANGELOG.md
-version: 3.13.0
+version: 3.15.0
```

Comment:
Replace the last line of the file with `- Fix 'maxmindLicenseKey' location in values.yaml`
(this is the changelog you see in https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx?modal=changelog)

Also, please squash the commits